.gitleaksignore

```diff
@@ -0,0 +1,5 @@
+1e4b4d2f9e11f8ea7a1cced14889e3e59a049f46:internal/state/secrets_test.go:private-key:42
```

Comment:
Is this autogenerated, or will we need to update this for any new test key we add?

Reply:
Unfortunately it's not autogenerated. We could add an exception for test files, maybe that's better?

Reply:
I think this is fine for now. If we add more test secrets later and this becomes a pain, we can revisit.